mqtt-app-dependencies/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.springframework.cloud.stream.app</groupId>
+	<artifactId>mqtt-app-dependencies</artifactId>
+	<version>1.2.1.BUILD-SNAPSHOT</version>
```

Comment:
I still strongly disagree going with with any other version other than 1.2.x as all others are 1.2.x as well. I would just be confusing.